src/main/webapp/js/components/interface/3dCanvas/ThreeDEngine.js

```diff
@@ -137,126 +137,115 @@ define(['jquery'], function () {
          */
         setupListeners: function () {
             var that = this;
-            var clientX,clientY;
             // when the mouse moves, call the given function
             this.renderer.domElement.addEventListener('mousedown', function (event) {
-            	clientX = event.clientX;
-            	clientY = event.clientY;
-            }, false);
-
-
-            this.renderer.domElement.addEventListener('mousemove', function (event) {
-            	that.mouse.x = (event.clientX / window.innerWidth) * 2 - 1;
-            	that.mouse.y = -(event.clientY / window.innerHeight) * 2 + 1;
-            	if (that.hoverListeners) {
-            		var intersects = that.getIntersectedObjects();
-            		for (var listener in that.hoverListeners) {
-            			if (intersects.length != 0) {
-            				that.hoverListeners[listener](intersects);
-            			}
-            		}
-            	}
-            }, false);
-            
-            this.renderer.domElement.addEventListener('mouseup', function (event) {
-            	if (event.target == that.renderer.domElement) {
-            		var x = event.clientX;
-            		var y = event.clientY;
-
-            		// If the mouse moved since the mousedown then don't consider this a selection
-            		if( x != clientX || y != clientY )
```

Comment:
This condition is quite critical to have as per first comment, differently we will end-up picking continuosly an item in the scene even when we want just rotate.